CMakeLists.txt

```diff
@@ -84,11 +84,16 @@ if (POLICY CMP0086)
   # UseSWIG honors SWIG_MODULE_NAME via -module flag
   cmake_policy(SET CMP0086 NEW)
 endif ()
+if (POLICY CMP0148)
+  # The FindPythonInterp and FindPythonLibs modules are removed
```

Comment:
are you sure ? we dont use FindPythonInterp anymore

Reply:
Yes, for that reason I've explicitly set the policy - that has been introduced with CMake-3.27 - to `NEW` behavior.

Reply:
See [CMake docs for CMP0148](https://cmake.org/cmake/help/latest/policy/CMP0148.html) for further reference.

Reply:
> The OLD behavior of this policy is for find_package(PythonInterp) and find_package(PythonLibs) to load the deprecated modules. The NEW behavior is for uses of the modules to fail as if they do not exist.
>
> This policy was introduced in CMake version 3.27. CMake version 3.27.0-rc1 warns when the policy is not set and uses OLD behavior. Use the [cmake_policy()](https://cmake.org/cmake/help/latest/command/cmake_policy.html#command:cmake_policy) command to set it to OLD or NEW explicitly.